modules/SofaGeneralSimpleFem/BeamFEMForceField.cpp

```diff
@@ -34,14 +34,15 @@ namespace component
 namespace forcefield
 {
 
-using namespace sofa::defaulttype;
-
+namespace _beamfemforcefield_
```

Comment:
Hi Erik,

Thanks for the good question, 

Using a namespace specific to a .h allows you to have private namespace. 
To make this clear that the namespace is private it has the same name as the one of the file and it use the _ _ (convention taken from python to indicate private vs public things). 

Now, why it is useful to have a private level of namespace...simply because it allow you to do
```cpp
namespace sofa {
namespace core {
namespace _myclass_private_ {
     using sofa:defaultype:Vec3;
     using namespace sofa::core::objectmode; 

     MyClass : public BaseObject 
     {
            void method(Vec3f&); 
     }
} /// namespace _myclass_private
using _myclass_private_::MyClass; 
} /// namespace core
} /// namespace sofa
```

This is invisible to the ones that use MyClass...so user can write sofa::blahblah::MyClass while keeping the lines ```cpp using namespace & using sofa::core::Vec3``` polluting the public namespace.